server/MovieFlicker/src/main/java/io/egen/rest/entity/Movie_Country.java

```diff
@@ -0,0 +1,49 @@
+package io.egen.rest.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity
+@Table(name="movie_country")
+public class Movie_Country {
+
+	@Id
+	@GenericGenerator(name="customUUID", strategy="uuid2")
+	@GeneratedValue(generator="customUUID")
+	private String id;
+	
+	private String country;
+	
+	@ManyToOne
+	private Movies movie;
```

Comment:
You could have used Country entity directly on Movie entity.
And same goes for other entities as well (Genre, Actor, Writer etc.)